lib/slack-hook-handler.js

```diff
@@ -31,22 +33,120 @@ SlackHookHandler.prototype.getIntent = function(slackUser) {
  * @param {Object} params HTTP body of the webhook request, as a JSON-parsed dictionary.
```

Comment:
You can document properties of an object by adding lines like:
```
@param {string} params.user_id The slack user's ID
```

This would make things clearer in this function I think.